javascript/ql/src/Statements/UseOfReturnlessFunction.ql

```diff
@@ -33,7 +33,9 @@ predicate benignContext(Expr e) {
   inVoidContext(e) or 
   
   // A return statement is often used to just end the function.
-  e = any(Function f).getAReturnedExpr()
+  e = any(Function f).getBody()
+  or
+  e = any(ReturnStmt r).getExpr()
```

Comment:
What does this change do?

Reply:
It makes sure that the `benignContext` predicate also flags return-statements outside functions. 

Such as: 
```HTML
<body onload="return voidFunction()">
```